Dockerfile

```diff
@@ -1,15 +1,15 @@
-FROM golang:1.9
+FROM golang:1.9-alpine
 
 COPY cmd/freegeoip/public /var/www
 
 ADD . /go/src/github.com/apilayer/freegeoip
-RUN \
-	cd /go/src/github.com/apilayer/freegeoip/cmd/freegeoip && \
-	go get -d && go install && \
-	apt-get update && apt-get install -y libcap2-bin && \
-	setcap cap_net_bind_service=+ep /go/bin/freegeoip && \
-	apt-get clean && rm -rf /var/lib/apt/lists/* && \
-	useradd -ms /bin/bash freegeoip
+WORKDIR /go/src/github.com/apilayer/freegeoip/cmd/freegeoip
+RUN apk update
+RUN apk add git libcap shadow
```

Comment:
`RUN apk add --no-cache git libcap shadow`
Removes the need to run `apk update` and to remove package lists after.

Reply:
Nice! Didn't know that.